docs/ops/state_backends.md

```diff
@@ -123,8 +123,7 @@ RocksDBStateBackend is currently the only backend that offers incremental checkp
 
 ## Configuring a State Backend
 
-State backends can be configured per job. In addition, you can define a default state backend to be used when the
-job does not explicitly define a state backend.
+State backends can be configured per job in code. In addition, you can define a default state backend in **flink-conf.yaml** that is used when the job does not explicitly define a state backend.
```

Comment:
huh...i just noticed the 2 subsections below that describe how to configure the default/per-job state backend. Aren't we just duplicating details here?

@alpinegizmo Do you have any input?